RPerformanceTracking/Private/_RPTSender.m

```diff
@@ -114,6 +114,16 @@ - (void)processLoop
         {
             index = [self sendWithStartIndex:index endIndex:idIndex];
         }
+        else
```

Comment:
if I understand correctly, the logic here is we don't want to send data every processLoop so we only send if count >= MIN_COUNT.